### ISSUE TITLE
method missing with ruby 2.3.0

### ISSUE THREAD

**Issue by jonathantribouharet** [heroku/heroku#1895]
Hi,
I just upgrade from ruby 2.2.3 to ruby 2.3.0 and now for each command with heroku i have the error:

```
/home/jonathan/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/specification.rb:2158:in `method_missing': undefined method `this' for #<Gem::Specification:0x55e498 heroku-3.42.35> (NoMethodError)
	from /home/jonathan/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/specification.rb:1057:in `find_active_stub_by_path'
	from /home/jonathan/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:64:in `require'
	from /home/jonathan/.rvm/gems/ruby-2.3.0@bim/gems/heroku-3.42.35/bin/heroku:11:in `<top (required)>'
	from /home/jonathan/.rvm/gems/ruby-2.3.0@bim/bin/heroku:23:in `load'
	from /home/jonathan/.rvm/gems/ruby-2.3.0@bim/bin/heroku:23:in `<main>'
	from /home/jonathan/.rvm/gems/ruby-2.3.0@bim/bin/ruby_executable_hooks:15:in `eval'
	from /home/jonathan/.rvm/gems/ruby-2.3.0@bim/bin/ruby_executable_hooks:15:in `<main>'
```

I use heroku-3.42.35.

**Comment by jonathantribouharet** [comment]
It's seem coming from `rubygems` https://github.com/rubygems/rubygems/issues/1420

**Comment by ransombriggs** [comment]
@jonathantribouharet I was not able to reproduce (but I use rbenv instead of rvm).  I will install rvm on my ubuntu vm and see if I can reproduce, in the meantime would it be possible for you to use 2.2.3 until this is fixed?

**Comment by jonathantribouharet** [comment]
Yes no problem, I already downgrade to ruby 2.2.3.

**Comment by ransombriggs** [comment]
@jonathantribouharet Could you please switch back to 2.3.0 and run `gem update heroku` and try again? I was unable to reproduce your exact error, but I have shipped a fix which should hopefully fix your problem. The version with the fix is in `3.42.37`, which you can verify via `gem list heroku`.

**Comment by jonathantribouharet** [comment]
Yes it works, thank you.
